onnxscript/rewriter/pattern.py

```diff
@@ -472,15 +479,20 @@ def __init__(
         attributes: dict[str, AttrPattern],
         outputs: Sequence[str | None],
         allow_other_attributes: bool | None,
+        _allow_other_inputs: bool | None,
```

Comment:
Is there a significance in prepending `_` for this arg?

Reply:
No, thanks for catching this.